website/content/docs/agent-and-proxy/agent/index.mdx

```diff
@@ -298,6 +298,18 @@ runtime metrics about its performance, the auto-auth and the cache status:
 | `vault.agent.cache.hit`          | Number of cache hits                                 | counter |
 | `vault.agent.cache.miss`         | Number of cache misses                               | counter |
 
+### IMPORTANT: `VAULT_ADDR` usage
+
+If the `VAULT_ADDR` environment variable on the Vault Agent instance, that value takes precedence over the value in the configuration file. The Vault Agent uses that to connect to Vault and this can create an infinite loop where the value of `VAULT_ADDR` is used to make a connection, and the Vault Agent ends up trying to connect to itself instead of the server.
```

Comment:
```suggestion
If you export the `VAULT_ADDR` environment variable on the Vault Agent instance, that value takes precedence over the value in the configuration file. The Vault Agent uses that to connect to Vault and this can create an infinite loop where the value of `VAULT_ADDR` is used to make a connection, and the Vault Agent ends up trying to connect to itself instead of the server.
```